src/pal/src/CMakeLists.txt

```diff
@@ -6,6 +6,14 @@ project(coreclrpal)
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
+if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
+    # Enable @rpath support for shared libraries.
+    set(MACOSX_RPATH ON)
+    if(CMAKE_VERSION VERSION_EQUAL 3.0 OR CMAKE_VERSION VERSION_GREATER 3.0)
+        cmake_policy(SET CMP0042 NEW)
```

Comment:
This is probably true for all CMake > 3.0 no?  I think Apple just barks about it because the default there in brew is 3.something.  This policy can probably go for all targets?